lib/onebox/engine.rb

```diff
@@ -151,8 +151,8 @@ def always_https?
 require_relative "layout_support"
 require_relative "file_type_finder"
 require_relative "engine/standard_embed"
-require_relative "engine/html"
-require_relative "engine/json"
+require_relative "engine/h_t_m_l"
```

Comment:
Hmm this file name is weird. Is there something we can do about this?

Reply:
Not if we want to follow rails/zeitwerk conventions, have `lib` in the autoload path and not add inflections :grimacing:

Reply:
Is there a reason why we want to avoid inflections? I think inflections are created specifically for this purpose.

Reply:
I don't think inflections are a bad thing and should be avoided. Specifically for this case, we are dealing with acronyms and it isn't possible for Rails to know all kind of acronyms which is why there is an API to register certain strings as acronyms.

Reply:
I agree with you, I’m just not aware enough on how we want to do things at Discourse and renaming stuff is a way to achieve the same result.